### PR TITLE
Added autocomplete for CPU

### PR DESCRIPTION
**Pull request by Aditi-1400**
- Added CPU autocomplete functionality
- Removed react-select due to delayed state change

### CODE REVIEW

**Comment by aparcar** [comment]
I tried this locally but receive the following error message, how did you test this?
image

File: form/src/App.js [hunk 1]
```diff
@@ -29,28 +36,27 @@ function App() {
...
+  //       onChange={async (e) => {
+  //         await props.onChange(e.value);
+  //       }}
+  //     ></Select>
+  //   );
+  // };
+
+  // const widgets = {
+  //   SelectWidget: CustomSelect,
+  // };
```
Comment (aparcar) [inline form/src/App.js]: You're mixing things within a single commit again. These changes are unrelated but you still commit them in a single run, resulting in one PR isn't compatible with the other. Use three PRs instead and if they depend on one another, mention that in the description.

File: form/src/App.js [hunk 2]
```diff
@@ -1,19 +1,26 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { render } from "react-dom";
 import Form from "@rjsf/bootstrap-4";
```
Comment (aparcar) [inline form/src/App.js]: Please start using a code linter which automatically makes sure the code style is the same everywhere. In this example there are partly spaces and partly not. Please create another PR which applies e.g. https://prettier.io/ everywhere.

File: form/src/App.js (hunk 1 above)
Comment (aparcar) [reply form/src/App.js]: Instead of *commenting* the code, please remove it. It'll still be in the git history to be looked at for future reference.

**Review by aparcar — CHANGES_REQUESTED** [review]

**Comment by Aditi-1400** [comment]
I am unable to understand why the validate metadata is failing! Also, I tested it simply by `yarn start`, are you still getting that error?

**Comment by aparcar** [comment]
Please make a clean copy of the git repository and try again.

**Comment by aparcar** [comment]
Tried again and with an updates `schema.json` it worked fine. Thanks! I'll update the CI to automatically add things to schema YAML files.

**Pull request merged**